src/views/RecipeItem.vue

```diff
@@ -5,7 +5,9 @@
     </div>
     <div class="fullsize-recipe-container-details">
       <a href="#">
-        <i class="fas fa-times"></i>
+        <router-link :to="{ name: 'Home' }"
```

Comment:
`<router-link>` is converted to `<a>` by vue.js under the hood.
There is not need for `<a/>` in `<a/>`. Remove it please